src/contact/components/EditInfo.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+
+export default class EditInfo extends React.Component {
+	constructor(props){
+		super(props);
+	}
+	componentDidMount(){
+		if(this.props.info === null){
+			this.props.actions.displayInfo(true);
+		} else {
+			this.props.actions.displayInfo(true,this.props.info);
+		}
+	}
+
+	addEmailField(){
```

Comment:
Unnecessary functions: addPhoneField and addPhoneField

Reply:
These functions are for adding secondary fields. (email & phone number)